File: Chapter2/FuzzyProteins/ScriptForGR/Perl/2-ResiduosFuzzy.pl
use strict;

open(lista,"/home/maria/Documentos/InterfaceRes/Fuzzy_NR");
#P07674 253 293 1R71_D

while(my $lista=<lista>){
	chomp $lista;
	my @sp=split " ",$lista;
	my @spchain= split "_",@sp[0];
	my @spres=split ";",@sp[1];
	#@spb[0]=lc(@spb[0]);
	open(res,">>/home/maria/Documentos/InterfaceRes/VPs/@spchain[0]/@spchain[0].ResPos");
	my $d=@spres;
	my $c=0;
	while($c<$d){
		print res "@spres[$c] @spchain[1]\n";
		$c++;
	}
}	
close(lista);

File: Chapter2/FuzzyProteins/ScriptForGR/Python/2-ResiduosFuzzy.py
with open("/home/maria/Documentos/InterfaceRes/Fuzzy_NR") as lista:
    for linea in lista:
        linea = linea.strip()
        sp = linea.split(" ")
        spchain = sp[0].split("_")
        spres = sp[1].split(";")
        # spb[0] = spb[0].lower()
        with open(f"/home/maria/Documentos/InterfaceRes/VPs/{spchain[0]}/{spchain[0]}.ResPos", "a") as res:
            d = len(spres)
            c = 0
            while c < d:
                res.write(f"{spres[c]} {spchain[1]}\n")
                c += 1
